documentation/src/components/Header.astro

```diff
@@ -48,13 +50,33 @@ const menuButtonVisible = !blogPage && !guidebook;
 					>GitHub</a
 				>
 			</div>
-			<button class="block rounded p-1 hover:bg-indigo-50" id="search-button">
+			<button
+				aria-label="Switch to dark theme"
+				id="light-theme-button"
+				class="block rounded p-1 hover:bg-bg-indigo-500/20"
```

Comment:
There's a number of instances of this change which I believe to be a mistake - `bg-bg-` is, to my knowledge, not valid tailwind, and the extra `bg-` likely makes this not work as intended.

Reply:
Thanks for catching this. I was using find and replace a lot and probably messed up one of the replaces.